lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -71,12 +74,52 @@ class IntegrationTestsManager {
     }
   }
 
-  void _runDriver() async {
-    startProcess(
-      './chromedriver/chromedriver',
-      ['--port=4444'],
-      workingDirectory: io.Directory.current.path
+  /// Clone flutter repository, use the youngest commit older than the engine
+  /// commit.
+  ///
+  /// Use engine/src/flutter/.dart_tools to clone the Flutter repo.
+  /// TODO(nurhan): Use git pull instead if repo exists.
+  Future<void> _cloneFlutterRepo() async {
+    // Delete directory if exists.
+    if(environment.engineDartToolDir.existsSync()) {
+      environment.engineDartToolDir.deleteSync();
+    }
+    environment.engineDartToolDir.createSync();
+
+    final int exitCode = await runProcess(
+      environment.cloneFlutterScript.path,
+      <String>[
+        environment.engineDartToolDir.path,
+      ],
+      workingDirectory: environment.webUiRootDir.path,
+    );
+
+    if (exitCode != 0) {
+      throw ToolException('ERROR: Failed to clone flutter repo. Exited with '
+          'exit code $exitCode');
+    }
+  }
+
+  Future<void> _enableWeb(String workingDirectory) async {
+    final int exitCode = await runProcess(
+      environment.flutterCommand.path,
+      <String>[
+        'config',
+        '--local-engine=host_debug_unopt',
```

Comment:
I can totally imagine we will one day forget to set `--local-engine` in some other place then scratch our heads looking into some obscure breakage. Why don't we have a utility next to `runProcess` that runs the `flutter` command and makes sure to always pass `--local-engine`?

Reply:
Can you add more details? I didn't get the suggestion.

Reply:
I meant we could have a function `runFlutter`, which is the same as `runProcess` but it automatically adds `--local-engine` to the list of arguments. Then all places that call `flutter` no longer need to worry about this argument.

Reply:
BTW, I think this argument doesn't have to come after the sub-command. You can do `flutter --local-engine=host_debug_unopt config`, which is the same as `flutter config --local-engine=host_debug_unopt`

Reply:
done.